cinema_service/settings.py

```diff
@@ -20,14 +25,13 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.getenv("SECRET_KEY", "your-default-secret-key")
+
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.getenv("DEBUG", "1") == "1"
```

Comment:
Ensure that the `DEBUG` environment variable is set to `0` in production to avoid exposing sensitive information. Currently, it defaults to `1`, which is suitable for development but not for production.